doc/nrf/glossary.rst

```diff
@@ -998,7 +998,7 @@ Glossary
       A string that identifies a project release.
 
    VPR Core
-      Pronounced "Viper Core", a :term:`Core` that is compatible with the RISC-V instruction set, meaning the industry-standard RISC-V development tools can be used.
+      Pronounced "Viper Core," a :term:`Core` that is compatible with the RISC-V instruction set, meaning the industry-standard RISC-V development tools can be used.
```

Comment:
```suggestion
      Pronounced "Viper Core", a :term:`Core` that is compatible with the RISC-V instruction set, meaning the industry-standard RISC-V development tools can be used.
```

Reply:
discussed offline, and vale's edit is correct. could leave it as is.

Reply:
RMF page 53/54